test/nodetool/test_ring.py

```diff
@@ -56,16 +63,17 @@ def format_size(v):
     return f'{v} bytes'
 
 
-@pytest.mark.parametrize("keyspace,host_status,host_state",
+@pytest.mark.parametrize("keyspace,resolve_ip,host_status,host_state",
                          [
-                             ('ks', 'live', 'joining'),
-                             ('ks', 'live', 'normal'),
-                             ('ks', 'live', 'leaving'),
-                             ('ks', 'live', 'moving'),
-                             ('ks', 'down', 'n/a'),
-                             ('', 'live', 'normal'),
+                             ('ks', False, 'live', 'joining'),
+                             ('ks', False, 'live', 'normal'),
+                             ('ks', False, 'live', 'leaving'),
+                             ('ks', False, 'live', 'moving'),
+                             ('ks', False, 'down', 'n/a'),
+                             ('', False, 'live', 'normal'),
+                             ('', True, 'live', 'normal'),
```

Comment:
Instead of a `True`/`False`, you can use `None`/`-r`/`--resolve-ip`, so that we can check that scylla-nodetool accepts both the short and long variant. It is enough to test just once with each.

Reply:
good point.

Reply:
i am using empty string instead of `None`. simpler this way.